rmw_fastrtps_cpp/src/rmw_client.cpp

```diff
@@ -103,196 +127,287 @@ rmw_create_client(
     }
   }
 
+  std::lock_guard<std::mutex> lck(participant_info->entity_creation_mutex_);
+
+  /////
+  // Find and check existing topics and types
+
+  // Create Topic and Type names
+  const service_type_support_callbacks_t * service_members;
+  const message_type_support_callbacks_t * request_members;
+  const message_type_support_callbacks_t * response_members;
+
+  service_members = static_cast<const service_type_support_callbacks_t *>(type_support->data);
```

Comment:
@MiguelCompany nit: since you have an `static_cast` on the right hand side, using `auto` would be OK e.g.:
```suggestion
  auto service_members = static_cast<const service_type_support_callbacks_t *>(type_support->data);
```
Same elsewhere.

Reply:
Fixed on f50efc984dfff024743c3ade1585a950be588b07